src/commitment_scheme/kzg10/key.rs

```diff
@@ -355,10 +279,88 @@ impl OpeningKey {
 
 #[cfg(test)]
 mod test {
+    use super::super::{AggregateProof, PublicParameters};
```

Comment:
That's a bit weird, can't we just go to `AggregateProof` and `PublicParameters` from `crate::` instead?